integration/js/pages/AppPage.js

```diff
@@ -551,7 +567,11 @@ class AppPage {
   }
 
   async isAuthenticating() {
-    return await this.driver.findElement(elements.authenticationFlow).isDisplayed();
+    return await this.driver.findElement(elements.authenticationFlow).isDisplayed() || await this.driver.findElement(elements.permisssionsFlow).isDisplayed();
+  }
```

Comment:
I think the check for permissions should be a separate method based on how it is used. 

If you prefer only one method then the method name needs to change. The method is now checking for authentication or device permissions dialog.

Reply:
I'll go ahead and change the method name to isLoadingDeviceFlow. I'd make them separate methods, but we only use the function in one place so not much reason to separate them.